File: assignedproblems/littlequiz.java
import java.util.Scanner;

public class littlequiz {
    public static void main( String[] args ) {
        Scanner keyboard = new Scanner(System.in);
        int ans0, ans1, ans2, count = 0;
        System.out.println("Are you ready for a quiz? You don't actually have a choice if you want this program to run.");

        System.out.println("What is the capital of Alaska? Enter in option numbers. \n1: Melbourne\n2:Anchorage \n3:Juneau \n");
        ans0 = keyboard.nextInt();

        if (ans0 == 3) {
            count ++;
            System.out.println("Correct");
        }
        else {
            System.out.println("WRONG");
        }

        System.out.println("Can you store the value \"cat\" in a variable type int? \n1:Yes\n2:No\n");
        ans1 = keyboard.nextInt();

        if (ans1 == 2) {
            System.out.println("Correct");
            count ++;
        }
        else {
            System.out.println("False");
        }

        System.out.println("What is 9+6/3 equal to? Input only integer responses\n");
        ans2 = keyboard.nextInt();

        if (ans2 == 11) {
            System.out.println("Corrrect");
            count ++;
        }
        else {
            System.out.println("WRONG");
        }
        System.out.println("You got " + count + " answers right.");
    }
}

File: assignedproblems/littlequiz.py
yes = input("Are you ready for a quiz? ")
yes = True
count = 0
print("Good! Here it is.")
ans0 = int(input("What is the capital of Alaska? Enter in option numbers. \n1: Melbourne\n2:Anchorage \n3:Juneau \n"))
if ans0 == 3:
    print("correct")
    count += 1
else:
    print("NO")
ans1 = input("Can you store the value \"cat\" in a variable type int? yes or no will sufice as answers.\n")
if ans1 == "no" or ans1 == "No":
    print("correct")
    count += 1
else:
    print("FALSE")
ans2 = int(input("What is 9+6/3 equal to? Input only integer responses\n"))
if ans2 == 11:
    print("Correct")
    count += 1
else:
    print("FALSE")
print("You got", count, "answers correct.")